postgres.go

```diff
@@ -72,28 +81,34 @@ func (d *PostgresDatabase) UnmarshalJSON(b []byte) error {
 
 	d.Created = (*time.Time)(p.Created)
 	d.Updated = (*time.Time)(p.Updated)
+	d.OldestRestoreTime = (*time.Time)(p.OldestRestoreTime)
 	return nil
 }
 
 // PostgresCreateOptions fields are used when creating a new Postgres Database
 type PostgresCreateOptions struct {
-	Label                 string                  `json:"label"`
-	Region                string                  `json:"region"`
-	Type                  string                  `json:"type"`
-	Engine                string                  `json:"engine"`
-	AllowList             []string                `json:"allow_list,omitempty"`
-	ClusterSize           int                     `json:"cluster_size,omitempty"`
-	Encrypted             bool                    `json:"encrypted,omitempty"`
-	SSLConnection         bool                    `json:"ssl_connection,omitempty"`
-	ReplicationType       PostgresReplicationType `json:"replication_type,omitempty"`
-	ReplicationCommitType PostgresCommitType      `json:"replication_commit_type,omitempty"`
+	Label       string   `json:"label"`
+	Region      string   `json:"region"`
+	Type        string   `json:"type"`
+	Engine      string   `json:"engine"`
+	AllowList   []string `json:"allow_list,omitempty"`
+	ClusterSize int      `json:"cluster_size,omitempty"`
+
+	Encrypted             bool                    `json:"encrypted,omitempty"`               // This field doesn't exist in DBaaS v2
+	SSLConnection         bool                    `json:"ssl_connection,omitempty"`          // This field doesn't exist in DBaaS v2
+	ReplicationType       PostgresReplicationType `json:"replication_type,omitempty"`        // This field doesn't exist in DBaaS v2
+	ReplicationCommitType PostgresCommitType      `json:"replication_commit_type,omitempty"` // This field doesn't exist in DBaaS v2
+
+	Fork *DatabaseFork `json:"fork,omitempty"`
 }
 
 // PostgresUpdateOptions fields are used when altering the existing Postgres Database
 type PostgresUpdateOptions struct {
-	Label     string                     `json:"label,omitempty"`
-	AllowList *[]string                  `json:"allow_list,omitempty"`
-	Updates   *DatabaseMaintenanceWindow `json:"updates,omitempty"`
+	Label       string                     `json:"label,omitempty"`
+	AllowList   *[]string                  `json:"allow_list,omitempty"`
+	Updates     *DatabaseMaintenanceWindow `json:"updates,omitempty"`
+	Type        string                     `json:"type,omitempty"`
+	ClusterSize int                        `json:"cluster_size,omitempty"`
```

Comment:
Looks like the `version` field is also a valid update request body field

Reply:
Must have missed this in the API doc. Added the `version` field.